### ISSUE TITLE
Transformer on WiderAttribute predicition 

### ISSUE THREAD

**Issue by abhigoku10** [Kevinz-code/CSRA#7]
@Kevinz-code  thanks for providing the source code and great work. I have few queries which is mentioned below 

1. The vit model predicts the attributes of the person right like male,trousers ... what is the accuracy your getting on this 
2. I looked into the VIT implementation and compared it with "rwightman " implementation , you have used the same implementation the only changes is in the classifier part ie MHA-->CSRA part right ??
3. Is CSRA customed to only VIT transformer can it be implemented with other transformers like CrossVIT 
4. The size of the custom image  dataset which i have is of range from 80*56 to 124*128 should i change the patchsize from 16 to 8/4 since resizing the small resolution creates the pixelation blur effect 
5. Didi u keep the same training params for all the 3 training models 

Please do share your thoughts Thansk in adavance

**Comment by Kevinz-code** [comment]
Hi, @abhigoku10 

1. The accuracy (mAP, to be precise) of our VIT model on Wider Attribute dataset is listed in Table 5 in our paper.

2. Yes

3. Theoretically CSRA can be applied to any Transformer structure, since the group of patch embeddings can be viewed as the feature tensor (like 7x7x2048 in ResNet-101), so CSRA can be added after the feature tensor. If you'd like to use other VIT like CrossVIT, please remember to use all the patch embeddings to be sent to CSRA part.

4. I am not sure whether reducing patch size could bring performance gain. I thought patch size wasn't a crucial factor of the final performance and so is the size of the image dataset. Still, you can try Multi-scale VIT like CrossViT to validate it.

5. What is the "3 training models" ?  The training paramters like learning rate, momentum, weight decay are the same over all datasets and models. About batch-size and resolution, we use 224x224 with batch size 64 on Wider Attribute, 448x448 with batch size 16 on VOC and MS-COCO.

Best,
